quill-finagle-mysql/src/main/scala/io/getquill/sources/finagle/mysql/TransactionalFinagleMysqlSource.scala

```diff
@@ -0,0 +1,14 @@
+package io.getquill.sources.finagle.mysql
+
+import com.twitter.finagle.exp.mysql.Client
+import com.twitter.util.Future
+import io.getquill.FinagleMysqlSourceConfig
+import io.getquill.naming.NamingStrategy
+
+class TransactionalFinagleMysqlSource[N <: NamingStrategy](config: FinagleMysqlSourceConfig[N], txClient: Client)
```

Comment:
I can't, [the dateTimezone again](https://github.com/juliano/quill/blob/8ac807f1f860cbebd734087140d8dae5cb607897/quill-finagle-mysql/src/main/scala/io/getquill/sources/finagle/mysql/FinagleMysqlSource.scala#L34)